src/vonage/video.py

```diff
@@ -15,11 +16,7 @@ def __init__(self, client):
         self._client = client
 
     def create_session(self, session_options={}):
-        params = {
-            'archiveMode': 'manual',
-            'p2p.preference': 'disabled',
-            'location': None
-        }
+        params = {'archiveMode': 'manual', 'p2p.preference': 'disabled', 'location': None}
```

Comment:
This should take  media mode, and then flip `p2p.preference` to enabled or disabled. We want to keep the original signature versus what the API actually takes, as the new parameter is a better representation of what is going on.

Reply:
This is how it works; the params object here is what's provided in the API request if the user doesn't specifically overwrite. Lines 24-35 do what you're describing, inspecting the `session_options` object for `MediaMode` and setting `p2p.preference` based on this. Although looking at this part more carefully has helped me find a Python gotcha which I'll fix